### PR TITLE
Template updates after feedback

### PR DESCRIPTION
**Pull request by choldgraf**
Some more feedback from OceanHackWeek!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-hub.yml
Comment (yuvipanda) [inline .github/ISSUE_TEMPLATE/new-hub.yml:104]:
```suggestion
        - Google OAuth (e.g., myemailaddress@2i2c.org)
        - ORCID
        - Other (may not be possible, please specify in comments)
```

**Comment by yuvipanda** [comment]
OK to merge once the merge conflict is resolved!

**Review by damianavila — APPROVED** [review]
LGTM (modulo the merge conflict are resolved).

**Comment by choldgraf** [comment]
merging this in since we've got approvals, thanks all :-)

**Pull request merged**
